participant/helpers.py

```diff
@@ -0,0 +1,11 @@
+from .models import Participant
+from exercise.models import Trial
+
+
+def create_participant(exercise, trial=None):
+    if trial is not None:
+        trial = Trial.objects.get(pk=trial)
```

Comment:
This will throw if the there's no `Trial` with the `given trial_id`.
Since we allow Trial to be None I don't think an exception should be raised here in case the Trial cannot be found.

Reply:
I'll change the search on the Trial object